ingestion/tests/unit/source_connection/test_hive_connection.py

```diff
@@ -0,0 +1,28 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+from unittest import TestCase
+
+from metadata.generated.schema.entity.services.connections.database.hiveConnection import (
+    HiveScheme,
+    HiveSQLConnection,
+)
+from metadata.utils.source_connections import get_connection_url
+
+
+class HiveConnectionTest(TestCase):
+    def test_connection_url(self):
+        expected_result = "hive://localhost:10000/default"
+        databricks_conn_obj = HiveSQLConnection(
+            scheme=HiveScheme.hive, hostPort="localhost:10000", database="default"
```

Comment:
Can we use some authOptions for the tests? 🙏

Reply:
@pmbrull should we keep all the connection urls in same file? test_source_url
cc: @ulixius9

Reply:
as you prefer. As the tests should be short enough I thought about having them in a single file, but we can split it out if needed